app/controllers/handlers/AddExerciseHandler.scala

```diff
@@ -4,12 +4,15 @@ import java.util.Calendar
 
 import controllers.Application._
 import forms.{ExerciseEntriesForm, ExerciseEntryFormInput}
-import models.{ExerciseEntries, ExerciseEntry}
+import models.{ExerciseEntries, ExerciseEntry, Users}
 import play.api.data.{Form, FormError}
 import play.api.mvc.{AnyContent, Request, Result}
 import utils.authentication.UserAuthenticator
+import utils.groupme.Bot
 
 object AddExerciseHandler {
+  lazy val bot = new Bot(sys.env("GROUPME_BOT_ID"), sys.env("GROUPME_URL"))
```

Comment:
Do you want the AddExerciseHandler to set up the Bot? You could put this in Global. I nice Scala way is to create a Scala helper object. Which is just an Object of the same name in the same file. You could lazily create the default Bot from there, and it acts like a static method.